test/unit/caches/test_local.py

```diff
@@ -300,6 +300,8 @@ def patch_get_single_entry(mocker):
 def test_get_encryption_materials(patch_get_single_entry):
     cache = build_lcmc()
 
+    sentinel.plaintext_length = int()
```

Comment:
`sentinel` objects are unique reference objects and we should not override them. Also, just for simplicity, rather than assigning "an int", let's just use an arbitrary integer value.
```suggestion
    plaintext_length = 42
```